pkg/kubefed2/delete/delete.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package delete
+
+import (
+	"io"
+
+	"github.com/golang/glog"
+	"github.com/kubernetes-sigs/federation-v2/pkg/kubefed2/util"
+	"github.com/spf13/cobra"
+)
+
+var (
+	delete_long = `
+		Deletes the API resources used to federate a type or a resource.
+		Use "kubefed2 disable" instead, if the intent is to only disable
+		propagation of the given type or a resource.
+
+		Current context is assumed to be a Kubernetes cluster hosting
+		the federation control plane. Please use the
+		--host-cluster-context flag otherwise.`
+
+	delete_example = `
+		# Delete all API resources associated with and used for propagation
+		of type Service.
+		kubefed2 delete type Service
+
+		# Delete all API resources associated with and used for propagation
+		of federated resource my-svc of type Service.
+		kubefed2 delete resource Service my-svc`
```

Comment:
I did not see the implementation code for delete resource. Is this a placeholder? Or I missed something? This message is misleading if we do not implement it I think.

Reply:
You are right, I did initially find it ok to include the details about `resource` in help and examples for the current commands; but examples specifically could be misleading for the immediate user.
Have removed the examples for `resource` from both `disable` and `delete`.